google-cloud-translate/src/main/java/com/google/cloud/translate/spi/DefaultTranslateRpc.java

```diff
@@ -85,16 +118,18 @@ private static TranslateException translate(IOException exception) {
   @Override
   public List<TranslationsResource> translate(List<String> texts, Map<Option, ?> optionMap) {
     try {
-      String targetLanguage =
-          firstNonNull(TARGET_LANGUAGE.getString(optionMap), options.getTargetLanguage());
-      final String sourceLanguage = SOURCE_LANGUAGE.getString(optionMap);
-      List<TranslationsResource> translations =
-          translate.translations()
-              .list(texts, targetLanguage)
-              .setSource(sourceLanguage)
-              .setKey(options.getApiKey())
-              .execute()
-              .getTranslations();
+      // TODO use POST as soon as usage of "model" causes error in non-whitelisted projects
+       String targetLanguage =
```

Comment:
1. The indentation is off (3 spaces instead of 2)
2. s/causes error/correctly reports an error/